components/rmrk/Create/CreateToken.vue

```diff
@@ -271,7 +268,7 @@ export default class CreateToken extends mixins(
             setTimeout(
               () =>
                 this.listForSale(
-                  createdNFTs.value,
+                  createdNFTs.value[0],
```

Comment:
Would the length of nfts list be greater than one?

Reply:
https://github.com/kodadot/nft-gallery/pull/6156#discussion_r1216477888